core/src/main/scala/cats/MonoidK.scala

```diff
@@ -38,3 +47,13 @@ import simulacrum.typeclass
       def combine(x: F[A], y: F[A]): F[A] = self.combine(x, y)
     }
 }
+
+trait CompositeMonoidK[F[_],G[_]]
+  extends MonoidK[λ[α => F[G[α]]]] {
+
+  implicit def F: MonoidK[F]
+  implicit def G: MonoidK[G]
```

Comment:
We don't actually use this `G` at all. The same is true for the current `SemigroupK` - if you drop the `SemigroupK[G]` implicit evidence, everything still compiles.

We can freely take a `SemigroupK[F]` and turn it into a `SemigroupK[λ[α => F[G[α]]]` with no requirements on `G` (it can even be a polykinded type like `Nothing` or `Any`!). I think we should probably drop the requirements on `G`. I'm also curious if another name would make more sense, since we aren't actually composing two `MonoidK` instances together. However, we are creating a `MonoidK` instance for the composite type, so it still may be a reasonable name.